src/Lorentz/Contracts/BaseDAO/FA2.hs

```diff
@@ -90,8 +92,8 @@ transfer = do
       swap
 
     checkSender
-      :: forall f. Address & Storage & f
-      :-> Address & Storage & f
+      :: forall f. Address : Storage pm : f
+      :-> Address : Storage pm : f
```

Comment:
What is the difference between `:` and `&`, since nothing else had changed

Reply:
I believe `&` will be deprecated. It is recommended to use `:`, from what I see, they serve the same purpose.